libs/cardano-ledger-api/test/Test/Cardano/Ledger/Api/State/QuerySpec.hs

```diff
@@ -70,6 +91,12 @@ spec = do
     committeeMembersStateSpec @ConwayEra
     committeeMembersStateSpec @DijkstraEra
 
+queryPoolStateSpec :: forall era. ShelleyEraTest era => Spec
+queryPoolStateSpec =
+  prop ("mkPState' works as expected - " <> eraName @era) $ \(ps :: PState era) ->
+    let nes = def & nesEsL . esLStateL . lsCertStateL . certPStateL .~ ps
+     in queryPoolState nes Nothing `shouldBe` mkPState' id ps
```

Comment:
This is not a very useful test.
You are essentially testing `mkPState' id ps === mkPState' id ps`
Could you please remove it completely or replace it with something more sensible